x/oracle/abci.go

```diff
@@ -56,6 +56,8 @@ func EndBlocker(ctx sdk.Context, k Keeper) {
 			// TODO - update tax-cap
 			// Set price to the store
 			k.SetLunaPrice(ctx, denom, mod)
+			k.AfterPriceUpdated(ctx, denom, mod)
```

Comment:
Name of function could be improved; perhaps "PriceUpdateCleanup", or consider decomposing the cleanup functions into identifiable chunks.

Reply:
Removed